public/app/features/alerting/unified/components/notification-policies/Policy.tsx

```diff
@@ -469,7 +466,7 @@ function MetadataRow({
         {contactPoint && (
           <MetaText icon="at" data-testid="contact-point">
             <span>
-              <Trans i18nKey="alerting.policies.metadata.delivered-to">Delivered to</Trans>
+              <Trans i18nKey="alerting.policies.metadata.delivered-to">Delivered to</Trans>{' '}
```

Comment:
Adding spaces so the text for all of the below is more like
`Delivered to foo` 
rather than
`Delivered tofoo`